app/scripts/controllers/modals/logoutModal.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+/* jshint unused: false */
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:LogoutModalController
+ * @description
+ * # LogoutModalController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('LogoutModalController', function ($location, $filter, $scope, $uibModalInstance, Constants) {
+    $scope.endTimestamp = moment().add(30, 'seconds');
+    var lastInteraction = Date.parse(localStorage.getItem('origin-web-console-last-interaction-timestamp'));
+    var countdownInterval = setInterval(function() {
+      var timeRemaining = $filter('countdownToTimestamp')($scope.endTimestamp);
+      $('.countdown').text(function() {
```

Comment:
this shouldn't be in here, it should be in the same block in app.js that handles the other date / time counters

Reply:
Not really sure if what would be the benefit of having it in `app.js`. Isn't it better to have the have it separated inside the modal controller as part of the interval, since we have to update the DOM with the actual countdown number every second ?